src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -135,6 +143,9 @@ private[query] class EventsByTagPublisher(
 
     case Cancel =>
       context.stop(self)
+
+    case tagWritten: String if tagWritten == tag =>
+      self ! Continue
```

Comment:
I think those are different concerns. Limiting pubsub on the sending side is in order to not spam the network. The actual materialized view delay depends on a lot of factors, i.e. partitions, size of data to be copied, etc. I actually considered addressing the latter by allowing the poll interval to start variable with an exponentially increasing delay, i.e. 50ms, 100ms, [...], 1s, and then stay at 1s.

I just pushed an update which simply throttles the pubsub, but immediately does send the first message. CI hasn't picked it up (yet?) though.